lib/crypto/tasks.py

```diff
@@ -4,15 +4,21 @@
 import shutil
 import zipfile
 
+from django.db.models import Q
+
 from celeryutils import task
 from lxml import etree
 
 import amo
 from versions.models import Version
 from lib.crypto.packaged import sign_file
+from versions.compare import version_int
 
 log = logging.getLogger('z.task')
 
+MIN_FF_VERSION = '4'
+MAX_FF_VERSION = '37'
```

Comment:
I find these confusing. Maybe `MIN_D2C_VERSION` AND `MIN_NOT_D2C_VERSION`? Sorry if I've complicated things.